CMakeLists.txt

```diff
@@ -43,6 +43,7 @@ add_compile_options(
     -Wmissing-declarations
     $<$<COMPILE_LANGUAGE:CXX>:-Wextra-semi>
     $<$<COMPILE_LANGUAGE:CXX>:-Wold-style-cast>
+    -Wno-invalid-constexpr
```

Comment:
This is just stray from my local needs. (Need to update xcode, the current one still doesn't work without this.).